.github/workflows/lint.yml

```diff
@@ -1,14 +1,18 @@
-name: Linting
+name: Lint & Validate
 
 on:
   pull_request_target:
     branches:
       - main
-  workflow_dispatch:
+    paths-ignore:
+      - "src/technologies/*.json"
+      - "src/categories.json"
+      - "src/groups.json"
+      - "src/images/icons/**"
```

Comment:
Why did we exclude these?

Most of our contributions are to these files and I'd like to lint them.